app/middleware/cache_headers.py

```diff
@@ -26,9 +26,12 @@ def __call__(self, request):
 
         # Code to be executed for each request/response after
         # the view is called.
-
-        if request.path in ['/checker', '/get-token']:
-            # never cache the /checker and /get-token endpoints.
+        # comparing the last separation by / in both strings
+        if request.path.split('/')[-1] in [
+            i.split('/')[-1] for i in
+            ['/checker', f'{settings.API_BASE}/get-token', f'/{settings.HEALTHCHECK_ENDPOINT}']
+        ]:
```

Comment:
Doesn't seems to me correct here, the split is not needed if we use the full path in the list. Also the path for healthcheck is missing the API_BASE.
I've also just found out that the `/metrics` endpoint in this list is missing, it could be corrected part of this PR.